redbot/core/core_commands.py

```diff
@@ -276,7 +276,7 @@ class Core(commands.Cog, CoreLogic):
     @commands.command(hidden=True)
     async def ping(self, ctx: commands.Context):
         """Pong."""
-        await ctx.send("Pong.")
+        await ctx.send(_("Pong."))
```

Comment:
This particular string needs to not go through i18n. If you want to suggest a different string such as the unicode character for a ping pong paddle, that's fine, but it has already been decided that this command specifically needs to do as little as possible to make it a reliable way to verify responsiveness. An additional comment can be added about this for future viewers if this is done.

Reply:
Yes I can understand that. I put it back as it was before.